src/view/lesson_environment.js

```diff
@@ -143,61 +166,63 @@ var LessonEnvironment = React.createClass({
 
   showButton: function() {
     if (!this.state.docked) {
-      return <div style={{width: "20px",
-                          float: "right",
-                          backgroundColor: '#03a9f4'}}>
-               <a href='#'
-                  onClick={this._openSidebar}
-                  style={{textDecoration: 'none', color: 'white'}}>
-                 C
-                 O
-                 M
-                 A
-                 N
-                 D
-                 O
-                 S
-               </a>
-             </div>;
+      return  <div style={this.styles.commandsButton}
+                   onClick={this._openSidebar}>
+                C<br/>
```

Comment:
Now that this bar has a fixed size, we can try using a CSS transform to write this vertical text. But that can come in another PR.